contracts/TokenRequest.sol

```diff
@@ -6,50 +6,70 @@ import "@aragon/os/contracts/common/SafeERC20.sol";
 import "@aragon/os/contracts/lib/token/ERC20.sol";
 import "@aragon/apps-token-manager/contracts/TokenManager.sol";
 import "./lib/UintArrayLib.sol";
+import "./lib/ArrayUtils.sol";
 
 /**
 * Expected use requires the FINALISE_TOKEN_REQUEST_ROLE permission be given exclusively to a forwarder. A user can then
 * request tokens by calling createTokenRequest() to deposit funds and then calling finaliseTokenRequest() which will be called
 * via the forwarder if forwarding is successful, minting the user tokens.
 */
-contract TokenRequest is AragonApp {
+  contract TokenRequest is EtherTokenConstant, AragonApp {
```

Comment:
The `EtherTokenConstant` is actually already in the `AragonApp` inherited by the `VaultRecoverable` which in inherited by `AragonApp`